.eslintrc.yml

```diff
@@ -2,6 +2,12 @@ extends: airbnb-base
 plugins:
   - import
 rules:
+  no-underscore-dangle:
+  - error
+  - allowAfterThis: true
+  space-before-function-paren:
+  - error
+  - named: always
```

Comment:
I like that one aswell, just didn't wanted to mess more with those files.
Will update so it is always `function addProperty(`